lib/jsonapi/plugs/query_parser.ex

```diff
@@ -7,9 +7,9 @@ defmodule JSONAPI.QueryParser do
   import JSONAPI.Utils.String, only: [underscore: 1]
 
   @moduledoc """
-  Implements a fully JSONAPI V1 spec for parsing a complex query string and
-  returning Elixir datastructures. The purpose is to validate and encode incoming
-  queries and fail quickly.
+  Implements a fully JSONAPI V1 spec for parsing a complex query string via the
+  `query_params` field from a `Plug.Conn` struct and returning Elixir datastructures.
```

Comment:
Annotation: I added some clarity to the documentation to emphasize that the this Plug looks at the `query_params` and not the `params`.